dask_expr/tests/test_groupby.py

```diff
@@ -34,9 +34,17 @@ def test_groupby_numeric(pdf, df, api, numeric_only):
     expect = getattr(pdf.groupby("x"), api)(numeric_only=numeric_only)
     assert_eq(agg, expect)
 
+    g = df.groupby("x")
+    agg = getattr(g, api)(numeric_only=numeric_only)["y"]
+
+    expect = getattr(pdf.groupby("x"), api)(numeric_only=numeric_only)["y"]
+    assert_eq(agg, expect)
```

Comment:
Ah - so this was actually failing before, because we were falling back to the "incorrect" `Reduction._simplify_up` logic. Correct?

Reply:
Failing for var, the others just didn't push anything through